packages/xstate-wallet/package.json

```diff
@@ -30,7 +30,7 @@
     "@xstate/react": "1.0.0-rc.3",
     "async-lock": "1.2.2",
     "dexie": "3.0.0",
-    "ethers": "4.0.45",
+    "ethers": "5.0.0-beta.187",
```

Comment:
we should probably use the same version between the wallet and the simple-hub to avoid having to download 2 versions of the package. The hub currently uses 5.0.0-beta.184